CI failure on Windows (R 4.1)

Not sure if this is a problem related to Rtools40 or 32-bit (or both?), but it seems the CI fails due to ragg.

https://github.com/tidyverse/ggplot2/actions/runs/17668841738/job/50215825055#step:5:4498
```
  In file included from webp_anim_dev.cpp:3:
  AggDeviceWebPAnim.h:3:10: fatal error: webp/encode.h: No such file or directory
   #include <webp/encode.h>
            ^~~~~~~~~~~~~~~
  compilation terminated.
  make: *** [C:/R/etc/i386/Makeconf:245: webp_anim_dev.o] Error 1
  ERROR: compilation failed for package 'ragg'
```

--- a/.github/workflows/R-CMD-check.yaml
+++ b/.github/workflows/R-CMD-check.yaml
@@ -66,10 +66,9 @@ jobs:
           cache-version: 3
           extra-packages: >
             any::rcmdcheck,
+            ragg@1.4.0,
             Hmisc=?ignore-before-r=4.2.0,
             quantreg=?ignore-before-r=4.3.0
-          packages: >
-            ragg@1.4.0
           needs: check
 
       - uses: r-lib/actions/setup-r-dependencies@v2

--- a/.github/workflows/R-CMD-check.yaml
+++ b/.github/workflows/R-CMD-check.yaml
@@ -58,6 +58,15 @@ jobs:
           http-user-agent: ${{ matrix.config.http-user-agent }}
           use-public-rspm: true
 
+      # TODO: remove this when R 4.6 is released
+      - name: Tweak for old Windows (R 4.1)
+        uses: r-lib/actions/setup-r-dependencies@v2
+        if: ${{ matrix.config.os == 'windows-latest' && matrix.config.r = 'oldrel-4' }}
+        with:
+          cache-version: 3
+          packages: >
+            ragg@1.4.0
+
       - uses: r-lib/actions/setup-r-dependencies@v2
         with:
           cache-version: 3

--- a/.github/workflows/R-CMD-check.yaml
+++ b/.github/workflows/R-CMD-check.yaml
@@ -61,7 +61,7 @@ jobs:
       # TODO: remove this when R 4.6 is released
       - name: Tweak for old Windows (R 4.1)
         uses: r-lib/actions/setup-r-dependencies@v2
-        if: ${{ matrix.config.os == 'windows-latest' && matrix.config.r = 'oldrel-4' }}
+        if: ${{ matrix.config.os == 'windows-latest' && matrix.config.r == 'oldrel-4' }}
         with:
           cache-version: 3
           packages: >

--- a/.github/workflows/R-CMD-check.yaml
+++ b/.github/workflows/R-CMD-check.yaml
@@ -64,10 +64,16 @@ jobs:
         if: ${{ matrix.config.os == 'windows-latest' && matrix.config.r == 'oldrel-4' }}
         with:
           cache-version: 3
+          extra-packages: >
+            any::rcmdcheck,
+            Hmisc=?ignore-before-r=4.2.0,
+            quantreg=?ignore-before-r=4.3.0
           packages: >
             ragg@1.4.0
+          needs: check
 
       - uses: r-lib/actions/setup-r-dependencies@v2
+        if: ${{ matrix.config.os != 'windows-latest' || matrix.config.r != 'oldrel-4' }}
         with:
           cache-version: 3
           extra-packages: >

